Libplanet.Tests/Blockchain/BlockChainTest.DelayedRenderer.cs

```diff
@@ -0,0 +1,227 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Blockchain.Renderers;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
+using Libplanet.Tests.Common;
+using Libplanet.Tests.Common.Action;
+using Serilog;
+using Serilog.Events;
+using Xunit;
+
+namespace Libplanet.Tests.Blockchain
+{
+    public partial class BlockChainTest
+    {
+        [Fact]
+        public async Task DelayedRendererInReorg()
```

Comment:
Why should these tests be written here, instead of `DelayedActionRendererTest`?

Reply:
I made the tests here because I thought that this test is more like an integration test using the `BlockChain`. Would it be better to move to `DelayedActionRendererTest`?

Reply:
Moved the tests to `DelayedActionRendererTest`.